The job **`mingw-check-2`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/15985589022/job/45089358993) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/45089358993) [(plain)](https://github.com/rust-lang/rust/commit/235dec179e6ea481735591e976c7c975c78b2a4f/checks/45089358993/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
    Checking rustc_pattern_analysis v0.0.0 (/checkout/compiler/rustc_pattern_analysis)
error: unresolved link to `Self::predicates_of`
    --> compiler/rustc_middle/src/query/mod.rs:789:5
     |
789  | /     /// Returns the explicitly user-written *predicates* of the definition given by `DefId`
790  | |     /// that must be proven true at usage sites (and which can be assumed at definition site).
791  | |     ///
792  | |     /// You should probably use [`Self::predicates_of`] unless you're looking for
793  | |     /// predicates with explicit spans for diagnostics purposes.
     | |________________________________________________________________^
...
2659 |   rustc_feedable_queries! { define_feedable! }
     |   -------------------------------------------- in this macro invocation
     |
     = note: the link appears in this line:
             
             You should probably use [`Self::predicates_of`] unless you're looking for
                                      ^^^^^^^^^^^^^^^^^^^^^
     = note: the struct `TyCtxtFeed` has no field or associated item named `predicates_of`
     = note: `-D rustdoc::broken-intra-doc-links` implied by `-D warnings`
     = help: to override `-D warnings` add `#[allow(rustdoc::broken_intra_doc_links)]`
     = note: this error originates in the macro `define_feedable` which comes from the expansion of the macro `rustc_feedable_queries` (in Nightly builds, run with -Z macro-backtrace for more info)
---
warning: build failed, waiting for other jobs to finish...
[RUSTC-TIMING] rustc_pattern_analysis test:false 0.843
[RUSTC-TIMING] rustc_ast_lowering test:false 0.947
[RUSTC-TIMING] rustc_trait_selection test:false 5.468
Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:00:39
  local time: Mon Jun 30 23:55:51 UTC 2025
  network time: Mon, 30 Jun 2025 23:55:51 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_macros/src/query.rs
+++ b/compiler/rustc_macros/src/query.rs
@@ -413,7 +413,6 @@ pub(super) fn rustc_queries(input: TokenStream) -> TokenStream {
                 "Query {name} cannot be both `feedable` and `eval_always`."
             );
             feedable_queries.extend(quote! {
-                #(#doc_comments)*
                 [#attribute_stream] fn #name(#arg) #result,
             });
         }

--- a/compiler/rustc_middle/src/query/mod.rs
+++ b/compiler/rustc_middle/src/query/mod.rs
@@ -439,7 +439,6 @@ rustc_queries! {
     query predicates_of(key: DefId) -> ty::GenericPredicates<'tcx> {
         desc { |tcx| "computing predicates of `{}`", tcx.def_path_str(key) }
         cache_on_disk_if { key.is_local() }
-        feedable
     }
 
     query opaque_types_defined_by(

--- a/compiler/rustc_mir_transform/src/coroutine/by_move_body.rs
+++ b/compiler/rustc_mir_transform/src/coroutine/by_move_body.rs
@@ -239,7 +239,7 @@ pub(crate) fn coroutine_by_move_body_def_id<'tcx>(
     body_def.explicit_predicates_of(tcx.explicit_predicates_of(coroutine_def_id));
     body_def.generics_of(tcx.generics_of(coroutine_def_id).clone());
     body_def.param_env(tcx.param_env(coroutine_def_id));
-    body_def.predicates_of(tcx.predicates_of(coroutine_def_id));
+    body_def.explicit_predicates_of(tcx.explicit_predicates_of(coroutine_def_id));
 
     // The type of the coroutine is the `by_move_coroutine_ty`.
     body_def.type_of(ty::EarlyBinder::bind(by_move_coroutine_ty));

